src/main/java/com/minecolonies/coremod/blocks/BlockPaperwall.java

```diff
@@ -65,6 +65,8 @@ public void registerItemBlock(final IForgeRegistry<Item> registry)
     /**
      * Get the MapColor for this Block and the given BlockState
      */
+    @NotNull
+    @Deprecated
     @Override
     public MapColor getMapColor(final IBlockState state)
     {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1133)